Add post about PyLadies at EuroPython 2023 (#80)
pyvec/blog.python.cz: content/2023-08-29_pyladies-byly-jsme-na-europythonu.md

Title: PyLadies & Pyvo - Byly jsme na EuroPythonu
Date: 2023-08-29 18:00:00
Author: Karolina Surma, Mia Bajic

Do Prahy letos poprvé doputovala konference EuroPython.
Mezi 17. 7. a 23. 7. 2023 jste tam mohli potkat velkou část české Python komunity.
EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak rozšířit sítě do mezinárodních vod.

S PyLadies a s Pyvem jsme to nemohly zmeškat!

![samolepky_pyladies]({static}/images/samolepky_pyladies_2023.jpg)

## Aktivity PyLadies

> S nápadem, že oslovíme organizátory EuroPythonu přišla **Monika Bláhová** z brněnských PyLadies.

Reviewer: ```suggestion
S nápadem, že oslovíme organizátory EuroPythonu, přišla **Monika Bláhová** z brněnských PyLadies.
```

> Vděčíme jí rozhýbání a finální rozsah prezentace PyLadies na EuroPythonu.

Reviewer: ```suggestion
Vděčíme jí za rozhýbání a finální rozsah prezentace PyLadies na EuroPythonu.
```

Původně byl v plánu pouze večerní sraz PyLadies a příznivců, z něhož se postupně vyklubaly další aktivity.
Realizace dopadla na výbornou díky nasazení, jaké do příprav vložila **Mia Bajić**. Díky!

![aktivity_Python_komunity_na_EuPy]({static}/images/czech_python_at_eupy.jpg)

### Hadí workshop

**17. 7. 2023**, hned na začátku konference, se konal celodenní [Hadí workshop](https://ep2023.europython.eu/pyladies-workshop)
(zmiňován již zde na blogu [v roce 2019](https://blog.python.cz/pyladies-snake-report) a [2020](https://blog.python.cz/Hadi-workshop-v-Brne]).
Ovšem letos to bylo poprvé **anglicky**.
Workshop opět nachystal a bravurně přednesl **Petr Viktorin** s výpomocí party **kouček a koučů** z Prahy, Plzně a Brna.
Někteří s námi koučovali vůbec poprvé, vítáme v komunitě :)

> ![snake_workshop]({static}/images/snake-workshop-eupy-2023.jpg)

Reviewer: V PR je ještě `content/images/thumbnails/snake_workshop_eupy_2023.jpg`, který je ještě navíc vzhůru nohama. Hádám, že  to není nějakej easter egg :upside_down_face:

Author: Diky! Nemohla jsem najit, kde se stala chyba :D

### PyLadies Social Event

**21. 7. 2023**, tentokrát na zakončení konference, jsme pozvaly všechny PyLadies a příznivce na [večerní společenský event](https://ep2023.europython.eu/pyladies-social-event).
Do Hospůdky na Hradbách dorazilo přes 80 účastnic a účastníků konference.
Akce sklidila velký úspěch.

## Pyvo
V Praze už přes 10 let každou třetí středu v měsíci probíhá pražské Pyvo a EuroPython se konalo přesně v tomto týdnu. Ve spolupráci se spolupořadateli EuroPythonu jsme naplánovaly společné Pyvo, abychom daly naší místní komunitě příležitost pokecat s účastníky EuroPythonu. Na Pyvo přišlo kolem 150 lidí a bylo to zatím největší a organizačně nejnáročnější Pyvo, co kdy bylo. Jsem velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.

## OSS stůl

> Česká komunita byla pozvaná k vystavení stolu v prostoru pro projekty Open Source.

Reviewer: ```suggestion
Česká komunita byla pozvaná k vystavení stolu v prostoru pro Open Source projekty.
```
(naopak mi to zní kostrbatě)

> Dohodli jsme se na společné propagaci PyLadies, meetupů Pyvo, PyData a letošní edice konference [PyCon CZ](https://cz.pycon.org/2023/) (již od 15. září!).

Reviewer: ```suggestion
Dohodly jsme se na společné propagaci PyLadies, meetupů Pyvo, PyData a letošní edice konference [PyCon CZ](https://cz.pycon.org/2023/) (již od 15. září!).
```
Ve zbytku textu je pro shodu přísudku použit podmět ženský (ať už jako vy dvě nebo nespecifikovaná ženská část komunity), ponechal bych ho tedy i tady.

Author: Zde je to o dohodě nás (PyLadies) se zbytkem komunity, proto změna formy.

Author: Po zvazeni jsem se to rozhodla prece jen prepsat na -y.

![OSS_stul_Python_komunity]({static}/images/oss_table_at_eupy_2023.jpg)

## Poděkování Pyvci

Díky grantu, který PyLadies udělil Pyvec, nezisková organizace naší komunity, jsme mohly nabídnout návštěvnicím a návštěvníkům kvalitní samolepky s naším logem ve čtyřech barvách.

> Budeme opět přítomné na konferenci PyCon CZ, samolepky využíjeme také na lokálních eventech všude, kde se budou konat aktivity pod naší hlavičkou.

Reviewer: Tohle zní trochu lidštěji:
```suggestion
Opět nás najdete na konferenci PyCon CZ, samolepky využíjeme také na lokálních eventech a všude, kde se budou konat aktivity pod naší hlavičkou.
```

> Title: PyLadies & Pyvo - Byly jsme na EuroPythonu

Reviewer 2: ```suggestion
Title: PyLadies & Pyvo – Byly jsme na EuroPythonu
```

Date: 2023-08-29 18:00:00
Author: Karolina Surma, Mia Bajic

Do Prahy letos poprvé doputovala konference EuroPython.
Mezi 17. 7. a 23. 7. 2023 jste tam mohli potkat velkou část české Python komunity.

> EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak rozšířit sítě do mezinárodních vod.

Reviewer 2: Tahle věta je divná. Je tam vazba *jak možnost, tak rozšířit*. Přidal bych tam jedno slovo navíc:

```suggestion
EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak šanci rozšířit sítě do mezinárodních vod.
```

S PyLadies a s Pyvem jsme to nemohly zmeškat!

![samolepky_pyladies]({static}/images/samolepky_pyladies_2023.jpg)

## Aktivity PyLadies

S nápadem, že oslovíme organizátory EuroPythonu, přišla **Monika Bláhová** z brněnských PyLadies.
Vděčíme jí za rozhýbání a finální rozsah prezentace PyLadies na EuroPythonu.
Původně byl v plánu pouze večerní sraz PyLadies a příznivců, z něhož se postupně vyklubaly další aktivity.

> Realizace dopadla na výbornou díky nasazení, jaké do příprav vložila **Mia Bajić**. Díky!

Reviewer 2: ```suggestion
Realizace dopadla na výbornou díky nasazení, které do příprav vložila **Mia Bajić**. Díky!
```

![aktivity_Python_komunity_na_EuPy]({static}/images/czech_python_at_eupy.jpg)

### Hadí workshop

**17. 7. 2023**, hned na začátku konference, se konal celodenní [Hadí workshop](https://ep2023.europython.eu/pyladies-workshop)

> (zmiňován již zde na blogu [v roce 2019](https://blog.python.cz/pyladies-snake-report) a [2020](https://blog.python.cz/Hadi-workshop-v-Brne]).

Reviewer 2: ```suggestion
(zmiňován již zde na blogu [v roce 2019](https://blog.python.cz/pyladies-snake-report) a [2020](https://blog.python.cz/Hadi-workshop-v-Brne])).
```

Ovšem letos to bylo poprvé **anglicky**.
Workshop opět nachystal a bravurně přednesl **Petr Viktorin** s výpomocí party **kouček a koučů** z Prahy, Plzně a Brna.

> Někteří s námi koučovali vůbec poprvé, vítáme v komunitě :)

Reviewer 2: ```suggestion
Někteří s námi koučovali vůbec poprvé, vítáme je v komunitě :)
```

![snake_workshop]({static}/images/snake-workshop-eupy-2023.jpg)

### PyLadies Social Event

**21. 7. 2023**, tentokrát na zakončení konference, jsme pozvaly všechny PyLadies a příznivce na [večerní společenský event](https://ep2023.europython.eu/pyladies-social-event).
Do Hospůdky na Hradbách dorazilo přes 80 účastnic a účastníků konference.
Akce sklidila velký úspěch.

## Pyvo
V Praze už přes 10 let každou třetí středu v měsíci probíhá pražské Pyvo a EuroPython se konal přesně v tomto týdnu.

> Ve spolupráci se pořadateli EuroPythonu jsme naplánovaly společné Pyvo, abychom daly naší místní komunitě příležitost pokecat s účastníky EuroPythonu.

Reviewer 2: ```suggestion
Ve spolupráci s pořadateli EuroPythonu jsme naplánovaly společné Pyvo, abychom daly naší místní komunitě příležitost pokecat s účastníky EuroPythonu.
```

> Na Pyvo přišlo kolem 150 lidí a bylo to zatím největší a organizačně nejnáročnější Pyvo, co jsme v našich dějinách organizovaly.

Reviewer 2: ```suggestion
Na Pyvo přišlo kolem 150 lidí a bylo to zatím největší a organizačně nejnáročnější Pyvo, jaké jsme v našich dějinách organizovaly.
```

> Jsem velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa, a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.

Reviewer 2: ```suggestion
Jsme velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.
```

Author: Zde mi přijde, že ta čárka je nutná

Reviewer: Souhlas: https://prirucka.ujc.cas.cz/?id=153#nadpis4

Jen bych opravil to `Jsem` na `Jsme` :)

Reviewer 2: Myslel jsem: _Jsme rády, že A a B._

Reviewer 2: S tím, že _a zároveň_ je slučovací.

Reviewer 2: Hlavní otázka asi je: chceš říct, že jste rády, že jste udělali promo? Nebo jste rádi patří jen k první části? V druhém případě bych začal Zároveň novou větu:

```suggestion
Jsme velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa. Zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.
```

Author: Díky, předěláno na dvě věty v dcf5b3b37b4b6d84c8d6a7359992b37051d0276a

## OSS stůl

Česká komunita byla pozvaná k vystavení stolu v prostoru pro Open Source projekty.
Dohodly jsme se na společné propagaci PyLadies, meetupů Pyvo, PyData a letošní edice konference [PyCon CZ](https://cz.pycon.org/2023/) (již od 15. září!).

![OSS_stul_Python_komunity]({static}/images/oss_table_at_eupy_2023.jpg)

## Poděkování Pyvci

Díky grantu, který PyLadies udělil Pyvec, nezisková organizace naší komunity, jsme mohly nabídnout návštěvnicím a návštěvníkům kvalitní samolepky s naším logem ve čtyřech barvách.

> Opět nás najdete na konferenci PyCon CZ, samolepky využíjeme také na lokálních eventech a všude, kde se budou konat aktivity pod naší hlavičkou.

Reviewer 2: ```suggestion
Příště nás najdete na konferenci PyCon CZ 23. Samolepky využíjeme také na lokálních eventech a všude, kde se budou konat aktivity pod naší hlavičkou.
```

### PyLadies Social Event

**21. 7. 2023**, tentokrát na zakončení konference, jsme pozvaly všechny PyLadies a příznivce na [večerní společenský event](https://ep2023.europython.eu/pyladies-social-event).
Do Hospůdky na Hradbách dorazilo přes 80 účastnic a účastníků konference.
Akce sklidila velký úspěch.

## Pyvo

> V Praze už přes 10 let každou třetí středu v měsíci probíhá pražské Pyvo a EuroPython se konalo přesně v tomto týdnu. Ve spolupráci se spolupořadateli EuroPythonu jsme naplánovaly společné Pyvo, abychom daly naší místní komunitě příležitost pokecat s účastníky EuroPythonu. Na Pyvo přišlo kolem 150 lidí a bylo to zatím největší a organizačně nejnáročnější Pyvo, co kdy bylo. Jsem velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.

Author: ```suggestion
V Praze už přes 10 let každou třetí středu v měsíci probíhá pražské Pyvo a EuroPython se konal přesně v tomto týdnu.
Ve spolupráci se spolupořadateli EuroPythonu jsme naplánovaly společné Pyvo, abychom daly naší místní komunitě příležitost pokecat s účastníky EuroPythonu.
Na Pyvo přišlo kolem 150 lidí a bylo to zatím největší a organizačně nejnáročnější Pyvo, co jsme v našich dějinách organizovaly.
Jsme velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa, a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.
```

Reviewer: ` spolupráci se spolupořadateli` bych upravil na ` spolupráci s pořadateli` - pokud tam je důraz na nějaké spolupořádání záměrně, tak `s ostatními pořadateli`.
